Basic-Car-Maintenance.xcodeproj/xcshareddata/xcschemes/Basic-Car-Maintenance.xcscheme

```diff
@@ -57,6 +57,8 @@
       buildConfiguration = "Debug"
       selectedDebuggerIdentifier = "Xcode.DebuggerFoundation.Debugger.LLDB"
       selectedLauncherIdentifier = "Xcode.DebuggerFoundation.Launcher.LLDB"
+      language = "de"
+      region = "DE"
       launchStyle = "0"
```

Comment:
Remove these
These are by default changing the language / region (when they should just stay as system defaults)

Reply:
sorry, added this for testing and accidental committed it. I will revert it

Reply:
fixed it